lib/doctor.ts

```diff
@@ -28,150 +28,207 @@ export class Doctor implements NativeScriptDoctor.IDoctor {
+				}),
+				this.processSysInfoItem({
+					item: !sysInfoData.cocoaPodsVer || !sysInfoData.isCocoaPodsUpdateRequired,
+					infoMessage: "CocoaPods update is not required.",
+					warningMessage: "CocoaPods update required.",
 					additionalInformation: `You are using CocoaPods version ${sysInfoData.cocoaPodsVer} which does not support Xcode ${sysInfoData.xcodeVer} yet.${EOL}${EOL}You can update your cocoapods by running $sudo gem install cocoapods from a terminal.${EOL}${EOL}In order for the NativeScript CLI to be able to work correctly with this setup you need to install xcproj command line tool and add it to your PATH.Xcproj can be installed with homebrew by running $ brew install xcproj from the terminal`,
 					platforms: [Constants.IOS_PLATFORM_NAME]
-				});
-			}
+				})
+			]);
 
 			if (sysInfoData.xcodeVer && sysInfoData.cocoaPodsVer) {
 				let isCocoaPodsWorkingCorrectly = await this.sysInfo.isCocoaPodsWorkingCorrectly();
-				if (!isCocoaPodsWorkingCorrectly) {
-					result.push({
-						warning: "WARNING: There was a problem with CocoaPods",
+				result = result.concat([
+					this.processSysInfoItem({
+						item: isCocoaPodsWorkingCorrectly,
+						infoMessage: "CocoaPods are configured properly.",
+						warningMessage: "There was a problem with CocoaPods",
 						additionalInformation: "Verify that CocoaPods are configured properly.",
-						platforms: [Constants.IOS_PLATFORM_NAME]
-					});
-				}
+						platforms: [Constants.IOS_PLATFORM_NAME],
+					})
+				]);
 			}
 
-			if (sysInfoData.cocoaPodsVer && semver.valid(sysInfoData.cocoaPodsVer) && semver.lt(sysInfoData.cocoaPodsVer, Doctor.MIN_SUPPORTED_POD_VERSION)) {
-				result.push({
-					warning: `WARNING: Your current CocoaPods version is earlier than ${Doctor.MIN_SUPPORTED_POD_VERSION}.`,
+			result = result.concat([
+				this.processSysInfoItem({
+					item: !sysInfoData.cocoaPodsVer || !semver.valid(sysInfoData.cocoaPodsVer) || !semver.lt(sysInfoData.cocoaPodsVer, Doctor.MIN_SUPPORTED_POD_VERSION),
+					infoMessage: `Your current CocoaPods version is newer than ${Doctor.MIN_SUPPORTED_POD_VERSION}.`,
+					warningMessage: `Your current CocoaPods version is earlier than ${Doctor.MIN_SUPPORTED_POD_VERSION}.`,
 					additionalInformation: "You will not be able to build your projects for iOS if they contain plugin with CocoaPod file." + EOL
-					+ `To be able to build such projects, verify that you have at least ${Doctor.MIN_SUPPORTED_POD_VERSION} version installed.`,
+						+ `To be able to build such projects, verify that you have at least ${Doctor.MIN_SUPPORTED_POD_VERSION} version installed.`,
 					platforms: [Constants.IOS_PLATFORM_NAME]
-				});
-			}
-
-			if (!sysInfoData.pythonInfo.isInstalled) {
-				result.push({
-					warning: `WARNING: Couldn't retrieve installed python packages.`,
+				}),
+				this.processSysInfoItem({
+					item: sysInfoData.pythonInfo.isInstalled,
+					infoMessage: "Successfully retrieve installed python packages.",
```

Comment:
I think a better message for this would be:
`"Python installed and configured correctly."`